charts/azuremonitor-containers/templates/omsagent-arc-k8s-crd.yaml

```diff
@@ -19,8 +19,16 @@ metadata:
   name: container-insights-clusteridentityrequest
   namespace: azure-arc
 spec:
-  audience: https://monitoring.azure.com/
+  {{- if eq (.Values.Azure.Cluster.Cloud | lower) "azurepubliccloud" }}
+  audience: https://monitor.azure.com/
```

Comment:
is this correct? monitoring to monitor?

Reply:
Yes, https://montor.azure.com/ is consolidated audience which works for both Log analytics and custom metrics. Before MSI/DCR mode, we only use the token for custom metrics and which had only support for https://monitoring.azure.com/ audience.  I worked with custom metrics to have the unified audience support so that same token can be used for AMCS & custom metrics. This change already taken care in the MDSD for LA.